### DISCUSSION TITLE
HEX color support for LuckPerms

### DISCUSSION THREAD

**Discussion by imparium-de** [Zrips/CMI#4864] [Feature Requests]
LuckPerms uses the `&#rrggbb` format for the color handling. It also works for BungeeTablistPlus so it would be nice when CMI could make use of that too.

**Comment by DrakePork** [comment]
You can use cmi hex support {#rrggbb} in luckperms if you have CMI as the ChatManager and it'll work.

**Reply by imparium-de** [reply]
Oh what you're right! I tried that multiple times yesterday and it didn't work. I guess something was wrong on our end and restarting the whole bungee network fixed it. Thank you!
